_sass/base/_typography.scss

```diff
@@ -1,12 +1,13 @@
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  font-family: 'Open Sans', sans-serif;
+*{
```

Comment:
I believe it's better to use "body" instead of "*", to make it easier to override this rule for specific descendants.

Reply:
Fair point